### PR TITLE
Added SAS Programming package

### PR DESCRIPTION
**Pull request by rpardee**
Thanks very much!

### CODE REVIEW

File: repository/s.json [hunk 1]
```diff
@@ -73,6 +73,19 @@
...
 		{
+			"name": "SAS Programming",
+			"description": "For using SAS Institute's Analytics & Data Management system." ,
+			"details": "https://github.com/rpardee/sas",
+			"labels": ["language syntax", "build-system", "snippets"],
+			"author": ["rpardee", "friedegg", "seemack"],
+			"releases": [
+			{
+				"sublime_text": ">=3000",
+				"tags": true
```
Comment (wbond) [inline repository/s.json]: This and the two lines above all should be indented one more level.

**Comment by wbond** [comment]
Snippets should use the tab character for indentation. If the user has specified indentation should happen via spaces, Sublime Text will convert the tabs to the appropriate number of spaces.

**Comment by wbond** [comment]
I believe you should be able to use the built-in built system error navigation. You need to specify the regex to search for as a setting on the view named `result_file_regex`. You can see an example at https://github.com/golang/sublime-build/blob/master/golang_build.py#L958.

**Comment by FichteFoll** [comment]
Or refer to some [docs](http://docs.sublimetext.info/en/latest/reference/build_systems/configuration.html#capturing-build-system-results).

**Comment by wbond** [comment]
@FichteFoll It would be nice if you guys killed http://sublimetext.info/docs/en/ and forward it to relevant info.

**Comment by FichteFoll** [comment]
Agree. cc @guillermooo

**Comment by rpardee** [comment]
Thanks much for the feedback!  I have fixed the snippets (and hopefully the indentation now too).  

I'm unclear on the error nav though--build results get written to a separate .log file on disk, rather than come back from stdout, say.  So that file gets opened independently & is not a "results view", I don't think.  (At any rate, if I open one of these guys, the command palette doesn't show a 'next error' command, and hitting F4 doesn't give me any error or warning in the console--just does nothing.)  

You buy that? 

Thanks again!

**Comment by FichteFoll** [comment]
Besides the logging output, this is pretty much exactly what the default build system target, the `exec` command, does. You can check its implementation in `Default/exec.py`. You can also refer to that for the `result_file_regex` setting.

**Comment by rpardee** [comment]
Very cool--thanks!  I'll have a look at exec.py & see what I can see.

**Pull request merged**